features/matchers/have_enqueued_job_matcher.feature

```diff
@@ -0,0 +1,75 @@
+Feature: have_enqueued_job matcher
+
+  The `have_enqueued_job` matcher is used to check if given ActiveJob job was enqueued.
+
+  Scenario: Checking job class name
+    Given active job integrated
```

Comment:
Since every Scenario has the `Given active job integrated` line, you could use a [Background](https://github.com/cucumber/cucumber/wiki/Background) to make it less repetitive.

I might also suggest the wording `Given active job integration` or `Given active job is available` - documentation-wise I think they are more-clear.